TeaDiary/Scripts/App/linkAttribute.js

```diff
@@ -0,0 +1,3 @@
+﻿$(document).on('click', '.link', function () {
```

Comment:
`linkAttribute.js` you say. But it's a class, isn't it?
Actually, now that I think about it. It makes more sense to actually make it an attribute and update your styles/logic accordingly, rather than to rename this file.
Think:
`<tag link href='...'/>`

It's much more obvious that `link` somehow extends the behavior.